src/hyp3_back_projection/utils.py

```diff
@@ -186,7 +166,7 @@ def download_orbit(granule_name: str, output_dir: Path) -> Path:
     Returns:
         Path to the downloaded orbit file
     """
-    orbit_path, _ = downloadSentinelOrbitFile(granule_name, str(output_dir), esa_credentials=get_esa_credentials())
+    orbit_path = str(fetch_for_scene(granule_name, str(output_dir)))
```

Comment:
`fetch_for_scene` accepts a `Path` object, and I like using the `dir=` keyword to be more explicit.
```suggestion
    orbit_path = str(fetch_for_scene(granule_name, dir=output_dir))
```